README.md

```diff
@@ -19,24 +19,51 @@ cda2fhir
 can be easily introduced by reusing the already existing section and entry transformers. Although the cda2fhir library expects C-CDA R2.1 compliant
 documents/entries, it has been tested as well with several older document instances compliant with earlier releases of C-CDA. The 
 [HAPI FHIR Validator](http://hapifhir.io/doc_validation.html) is also integrated for automated validation of the generated FHIR resources.
 
-All the mappings implemented between CDA artifacts and FHIR resources, data types and value sets are documented in this sheet:
-[CDA to STU3 Mapping](https://docs.google.com/spreadsheets/d/e/2PACX-1vSVo_OoZXDxaLHwllt7EfOtePIY8EpiphlCrNssOrq7rvzVgdO361eTXQ96xiHugTJvY12J_-zibpeJ/pubhtml#)
-
-[Model Driven Health Tools (MDHT)](https://projects.eclipse.org/projects/modeling.mdht) is used for CDA manipulation and
-[HAPI](http://hapifhir.io/) is used for FHIR manipulation. The current implementation produces STU3 resources.
+##Latest Updates
+The original cda2fhir library created by [SRDC](https://github.com/srdc/cda2fhir) mapped C-CDA resources to FHIR DSTU2-compliant resources. Amida has created this fork of this library, incorporating the work of [CarthageKing](https://github.com/CarthageKing/cda2fhir), to instead map C-CDA resources to FHIR STU3-compliant resources. [Model Driven Health Tools (MDHT)](https://projects.eclipse.org/projects/modeling.mdht) is used for CDA manipulation and
+[HAPI](http://hapifhir.io/) is used for FHIR manipulation. This version of cda2fhir currently supports the following C-CDA Section to Resource mappings:
+
+|C-CDA Section    | FHIR Resource(s)   |
+|------------------|-----------------|
+|Medications           |MedicationStatement, MedicationRequest, MedicationDispense, Medication|
+|Procedures           |Procedure|
+|Immunizations |Immunization, Medication|
+|Results| DiagnosticReport, Observation|
+|Vital Signs| Observation|
+|Problems (Conditions)| Condition|
+|Allergies and Intolerances| AllergyIntolerance|
+|Encounters| Encounter|
+
+In addition to the above mappings, cda2fhir also uses and supports the Patient, Practitioner, PractitionerRole, Organization, Device, DocumentReference, Composition, and Provenance FHIR Resources.
+
+**This version of the cda2fhir library implements several additional features not present in the SRDC version. These include:**
+
+* cda2fhir is now capable of generating "transactional" bundles.
+* cda2fhir now supports the generation of Provenance objects, optionally taking in an Identifier resource and string representation of the source file to generate the accompanying Device and DocumentReference resources respectively.
+* Bundles now de-duplicate against themselves for certain resources; this is done to prevent duplicate resources from being created in a FHIR server, and works together with the ifNoneExist parameters.
+  * Medications are de-duplicated based on their encoding.
+  * Organizations and Practitioners are deduplicated based on identifier.
```

Comment:
Also organization identifier for medications.

Reply:
added a note under the meds bullet.